vault/core.go

```diff
@@ -3551,3 +3556,36 @@ func (c *Core) GetHCPLinkStatus() (string, string) {
 
 	return status, resourceID
 }
+
+// ListenerAddresses provides a slice of configured listener addresses
+func (c *Core) ListenerAddresses() ([]string, error) {
+	addresses := make([]string, 0)
+
+	conf := c.rawConfig.Load()
+	if conf == nil {
+		return nil, fmt.Errorf("failed to load core raw config")
+	}
+
+	listeners := conf.(*server.Config).Listeners
+	if lns == nil {
```

Comment:
I am seeing a millions of GH warnings about an undefined `lns` :)

Reply:
@ccapurso I think this still needs to be addressed

Reply:
Oh shoot! I fixed that but apparently I didn't commit and push. Thanks!

Reply:
Actually, it looks like I did and the build works locally. I'll dig in and try to figure that out.